README.md

```diff
@@ -23,7 +23,8 @@ Version matrix:
 
 | Kubernetes Version | cri-tools Version | cri-tools branch |
 |--------------------|-------------------|------------------|
-| 1.17.x             | v1.17.0           | master           |
+| 1.18.x             | v1.18.0           | master           |
+| 1.17.x             | v1.17.0           | release-1.17     |
```

Comment:
Ref: https://github.com/kubernetes-sigs/cri-tools/issues/558#issuecomment-563984791

Reply:
let's use master branch and merge 1.16/1.17 together? Since CRI API doesn't change recently, I think we don't need to maintain a new release branch. WDYT?

Reply:
Yes sounds good, changed it as suggested. :+1: